src/lamplib/src/genny/tasks/create-new-actor.sh

```diff
@@ -338,6 +338,11 @@ void ${actor_name}::run() {
     // Actors so that all Actors start and end at the right time.
     //
     for (auto&& config : _loop) {
+        //
+        // This will print out some information about the actor beginning execution.
+        // This will occur for every actor thread, and will print out the actor type,
+        // name and id. Debug mode is used to not spam logs for high thread actors.
+        BOOST_LOG_TRIVIAL(debug) << "Starting " << this->actorInfo() << " execution";
```

Comment:
Worth noting that I think the rate limiter is not accounted for here. I.e. an actor may print "Starting ... execution", then be rate limited and not act for a very long time, since rate limiting is part of the inner actor iteration.

Not that I think there's anything that can be done about that at this level, just probably worth noting here.